starport/services/chain/chain.go

```diff
@@ -214,8 +223,12 @@ func (c *Chain) StoragePaths() (paths []string, err error) {
 	return paths, nil
 }
 
+// Config returns the config of the chain
 func (c *Chain) Config() (conf.Config, error) {
-	path, err := conf.Locate(c.app.Path)
+	if c.options.ConfigFile != "" {
+		return conf.ParseFile(c.options.ConfigFile)
```

Comment:
https://github.com/tendermint/starport/pull/761#pullrequestreview-591864052 can eliminate this change as well.